plugins/com.google.cloud.tools.eclipse.swtbot/src/com/google/cloud/tools/eclipse/swtbot/SwtBotWorkbenchActions.java

```diff
@@ -184,7 +184,7 @@ public static void resetWorkbench(SWTWorkbenchBot bot) {
    * Reimplementation of {@link SWTWorkbenchBot#closeAllShells()} that does not close the internal
    * Eclipse Workbench limbo shell thus avoiding Eclipse bug 511729.
    */
-  private static void closeAllShells(final SWTWorkbenchBot bot) {
+  public static void closeAllShells(final SWTWorkbenchBot bot, final boolean forceKill) {
```

Comment:
optionally since boolean parameters aren't very object oriented, you could replace this with two methods: closeAllShells and killAllShells.